pkg/controllers/common/utils.go

```diff
@@ -17,6 +17,10 @@ import (
 	servicecommon "github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
 )
 
+const (
+	NumberConciler = 8
```

Comment:
We can define some constant values(maybe in common/types.go?) like MaxConcurrentReconciles in general, and specific ones for SubnetPort/Pod? Or we can wait for system test result before we distinguish between different resources.